theano/tensor/sort.py

```diff
@@ -350,14 +337,11 @@ class TopKOp(theano.Op):
     def __init__(
             self,
             axis=-1,
-            return_indices=False,
```

Comment:
You need to keep them in the \__init__ method, but you don't need to keep them in the topk function. We should never modify the \__props__ of an instance of an Op.